docs/further.md

```
@@ -32,13 +32,17 @@ With these changes, your job will be scheduled only on GPU-enabled nodes, and th
     gpu_manufacturer="nvidia",  # Allowed values: "nvidia" or "amd"
-    machine_type="n1-standard-16" 
+    machine_type="n1-standard-16", 
+    scale=0                     # Optional (default=1)
 ```
 - `gpu`: the number of GPUs to be requested
     - note: This currently only works for multiple GPUs within a single node. The current implementation cannot handle requesting multiple nodes with GPUs.
 - `gpu_manufacturer`: Specifies the GPU vendor. Use "nvidia" for NVIDIA GPUs or "amd" for AMD GPUs.
 - `machine_type`: machine type for the GPU enabled node. This is NOT the GPU type.
     - https://cloud.google.com/compute/docs/general-purpose-machines
+-  `scale`: variable allows us to conditionally include resource (GPU, threads, memory, etc) limits - those limits being equal to the resource requests.
+    - If `scale=1`(the default), we omit the limits entirely. This is how the plugin currently operates and will allow the pods to scale up as needed.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)